src/eduid_webapp/security/views/u2f.py

```diff
@@ -68,6 +68,8 @@ def bind(user, version, registration_data, client_data, description=''):
 
     cert = x509.load_der_x509_certificate(der_cert, default_backend())
     pem_cert = crypto.dump_certificate(crypto.FILETYPE_PEM, cert)
+    if six.PY3:
```

Comment:
What you're trying to do is always have `pem_cert` to be of a `six.string_types` type, so
```python
if not isinstance(pem_cert, six.string_types):
    pem_cert = pem_cert.decode("utf-8")
```

Reply:
Good point!

Reply:
To be picky, maybe

`
if isinstance(pem_cert, six.binary_types):
    pem_cert = pem_cert.decode("utf-8")
`
should have been used, to make sure there is a decode() method on pem_cert instead of assuming it is bytes just because it is not string_types... But on the other hand, we might prefer a crash if pem_cert is None for example. I'm going to merge this but just wanted to bring this up for consideration.